install.sh

```diff
@@ -143,7 +143,7 @@ if [[ "$answer" -eq 1 ]]; then
     sudo apt-get install -qq -y ripgrep
 fi
 
-sudo apt-get install -qq -y {curl,wget,stow,build-essential,unzip,tree,dialog,bc}
+sudo apt-get install -qq -y curl wget stow build-essential unzip tree dialog bc fakeroot
```

Comment:
`fakeroot` package hasn't been updated since 2014. Highly unlikely people will have an outdated version.

And `bc` is even older, it was last updated in 2011

Reply:
We also don't use `dialog` since 1.5 release